diff --git a/include/configs/colibri_t30.h b/include/configs/colibri_t30.h
--- a/include/configs/colibri_t30.h
+++ b/include/configs/colibri_t30.h
@@ -24,8 +24,8 @@
 #define CONFIG_DISPLAY_BOARDINFO_LATE	/* Calls show_board_info() */

 /* Board-specific serial config */
-#define CONFIG_TEGRA_ENABLE_UARTA
-#define CONFIG_SYS_NS16550_COM1		NV_PA_APB_UARTA_BASE
+#define CONFIG_TEGRA_ENABLE_UARTB
+#define CONFIG_SYS_NS16550_COM1		NV_PA_APB_UARTB_BASE

 #define CONFIG_MACH_TYPE		MACH_TYPE_COLIBRI_T30

@@ -134,7 +134,7 @@

 #define BOARD_EXTRA_ENV_SETTINGS \
 	"boot_file=zImage\0" \
-	"console=ttyS0\0" \
+	"console=ttyHS3\0" \
 	"defargs=core_edp_mv=1300 usb_high_speed=1 user_debug=30\0" \
 	"dfu_alt_info=" DFU_ALT_EMMC_INFO "\0" \
 	EMMC_BOOTCMD \



