diff --git a/recipes-kernel/linux/linux-toradex_git.bb b/recipes-kernel/linux/linux-toradex_git.bb
index 66826cd..2223a7c 100644
--- a/recipes-kernel/linux/linux-toradex_git.bb
+++ b/recipes-kernel/linux/linux-toradex_git.bb
@@ -23,7 +23,11 @@ config_script () {
 #    #example change to the .config
 #    #sets CONFIG_TEGRA_CAMERA unconditionally to 'y'
 #    sed -i -e /CONFIG_TEGRA_CAMERA/d ${B}/.config
-#    echo "CONFIG_TEGRA_CAMERA=y" >> ${B}/.config
+    echo "CONFIG_CAN=y" >> ${B}/.config
+    echo "CONFIG_CAN_RAW=y" >> ${B}/.config
+    echo "CONFIG_CAN_BCM=y"  >> ${B}/.config
+    echo "CONFIG_CAN_DEV=y" >> ${B}/.config
+    echo "CONFIG_CAN_MCP251X=y"  >> ${B}/.config
     echo "dummy" > /dev/null
 }
 
